CHANGELOG.md

```diff
@@ -3315,6 +3315,11 @@ This is the last release to support building with Swift 4.0 and Swift 4.1.
 
 #### Enhancements
 
+* Add `period_spacing` opt-in rule that checks periods are not followed
+  by 2 or more spaces in comments.
```

Comment:
Funnily, entries in the CHANGELOG must end with 2 spaces so that they are separated from the links following in the rendered document:
```suggestion
  by 2 or more spaces in comments.  
```

Reply:
Now that weird comment I got before adding the entry to the changelog makes way more sense.